requirements/secure.txt

```diff
@@ -0,0 +1,6 @@
+-r ./dev.txt
```

Comment:
Can you explain why `dev.txt` dependencies are included in this file?

(as opposed to this being a totally separate flag)